css-grid-classes/styles.css

```diff
@@ -0,0 +1,24 @@
+.container{
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+.row{
+  display: flex;
+}
+.column-half{
+  width: 50%;
+  display: flex;
+}
+.column-third{
+  width: calc(100% / 3);
+}
+.link, .yoshi{
```

Comment:
you should not be setting height on the images. the width should dictate their height, so you should be setting the correct width on all of the images!